features/step_definitions/comment_steps.rb

```diff
@@ -103,5 +103,47 @@
 
 When /^I delete the comment$/ do
   step %{I follow "Delete" within ".odd"}
-    step %{I follow "Yes, delete!"}
+  step %{I follow "Yes, delete!"}
 end
+
+Given(/^the moderated work "([^\"]*?)" by "([^\"]*?)"$/) do |work, user|
+  step %{I am logged in as "#{user}"}
+  step %{I set up the draft "#{work}"}
+  check("work_moderated_commenting_enabled")
+  step %{I post the work without preview}
+end
+
+Then /^comment moderation should be enabled on "([^\"]*?)"/ do |work|
+  w = Work.find_by_title(work)
+  assert w.moderated_commenting_enabled?
+end
+
+Then /^comment moderation should not be enabled on "([^\"]*?)"/ do |work|
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.